model/project_ref.go

```diff
@@ -921,6 +921,47 @@ func FindMergedProjectRef(identifier string, version string, includeProjectConfi
 	return pRef, nil
 }
 
+// GetNumberOfEnabledProjects returns the current number of enabled projects on evergreen.
+func GetNumberOfEnabledProjects() (int, error) {
+	return countEnabledProjects("", "")
+}
+
+// GetNumberOfEnabledProjectsForOwnerRepo returns the number of enabled projects for a given owner/repo.
+func GetNumberOfEnabledProjectsForOwnerRepo(owner, repo string) (int, error) {
+	if owner == "" || repo == "" {
+		return 0, errors.New("owner and repo must be specified")
+	}
+	return countEnabledProjects(owner, repo)
+}
+
+func countEnabledProjects(owner, repo string) (int, error) {
+	pipeline := projectRefPipelineForValueIsBool(ProjectRefEnabledKey, RepoRefEnabledKey, true)
+	if owner != "" && repo != "" {
+		// Check owner and repos in project ref or repo ref.
+		query := bson.M{
+			"$or": []bson.M{
+				{ProjectRefOwnerKey: owner, ProjectRefRepoKey: repo},
+				{"repo_ref": bson.M{
+					"$elemMatch": bson.M{RepoRefOwnerKey: owner, RepoRefRepoKey: repo},
```

Comment:
Owner/Repo isn't supposed to need to be inherited from the repo; all projects explicitly define it (this is why in the projectRef these fields aren't pointers).

Reply:
Should be able to just use https://github.com/evergreen-ci/evergreen/blob/f4f422a9c06a80fa8069d622d943cfb7aeb514f2/model/project_ref.go#L1326

Reply:
done